KsApi/Service+RequestHelpers.swift

```diff
@@ -24,6 +25,35 @@ extension Service {
     .flatMap(self.decodeModel)
   }
 
+  func request<M: Decodable>(_ route: Route) -> AnyPublisher<M, ErrorEnvelope> {
```

Comment:
This is a fairly faithful line-for-line request of the same `request` method above.